src/assets/angular-tour.scss

```diff
@@ -0,0 +1,138 @@
+//colors
+$background_color: #1c252e;
+$white: #fff;
+$text_color: #cbd0d4;
+$button_color: #555;
+$button_hover_color: #777;
+$close_button_hover: #eee;
+$black_60: rgba(0, 0, 0, 0.6);
+
+
+%base-button {
+  background-color: $button_color;
+  transition: background-color 300ms ease-out;
+  cursor: pointer;
+  font-family: inherit;
+  font-weight: 700;
+  line-height: 1;
+  position: relative;
+  text-decoration: none;
+  text-align: center;
+  display: inline-block;
+  font-size: .8em;
```

Comment:
This is a little small, it comes across to me as visually cramped:
![image](https://cloud.githubusercontent.com/assets/442115/10531835/83352c64-7384-11e5-8f7b-1249532af1fa.png)

I might remove this rule, the button isn't so much bigger without the smaller font that I think it outweighs the legibility compromise